tgui/packages/tgui/interfaces/NifPanel.js

```diff
@@ -0,0 +1,204 @@
+import { useBackend, useLocalState } from '../backend';
+import { Window } from '../layouts';
+import { Box, Dropdown, LabeledList, ProgressBar, Section, Button, Input, BlockQuote, Flex, Collapsible, Table } from '../components';
+import { TableCell, TableRow } from '../components/Table';
+
+export const NifPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { linked_mob_name, loaded_nifsofts, max_nifsofts, max_power, current_theme } = data;
+  const [settingsOpen, setSettingsOpen] = useLocalState(
+    context,
+    settingsOpen,
```

Comment:
```suggestion
    'settingsOpen',
```
It's a string key. If this works at all, the system is coincidentally OK with a null key once.